src/widget.cpp

```diff
@@ -1571,10 +1571,14 @@ std::string widget::layout( const avatar &ava, unsigned int max_width, int label
         int row_num = 0;
         // For multi-line widgets, each line is separated by a '\n' character
         while( ( strpos = shown.find( '\n' ) ) != std::string::npos && row_num < _height ) {
+            if( _var == widget_var::body_graph || _var == widget_var::overmap_text ) {
+                //Multiline widgets become completely unreadable without spaces and destroy other widgets. Forcing an override here.
+                manual_align = false;
+            }
```

Comment:
```suggestion
            //Multiline widgets become completely unreadable without spaces and destroy other widgets. Forcing an override here.
            manual_align = false;
```
Only multiline widgets will encounter this block here. No need to check for specific ones